dev-embedded/sunxi-tools/sunxi-tools-1.4.2.ebuild

```diff
@@ -0,0 +1,56 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="8"
+
+inherit toolchain-funcs
+
+if [[ ${PV} = *9999* ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/linux-sunxi/sunxi-tools"
+else
+	KEYWORDS="~amd64"
+	SRC_URI="https://github.com/linux-sunxi/sunxi-tools/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+fi
+
+DESCRIPTION="A collection of command line tools for ARM devices with Allwinner SoCs"
+HOMEPAGE="https://linux-sunxi.org/Main_Page"
+
+LICENSE="GPL-2"
+SLOT="0"
+
+RDEPEND="acct-group/plugdev
+	virtual/libusb:1
+	virtual/udev"
+
+DEPEND="${RDEPEND}
+	virtual/pkgconfig"
```

Comment:
BDEPEND for virtual/pkgconfig please (doesn't belong in DEPEND)

Reply:
I just copied 1.4.1. Don't blame me :)